src/Form/Control/Lookup.php

```diff
@@ -147,6 +148,9 @@ protected function init(): void
         $this->initQuickNewRecord();
 
         $this->settings['forceSelection'] = false;
+
+        $this->callback = \atk4\ui\Callback::addTo($this);
```

Comment:
yes, this makes sense

Reply:
No. Move from renderView to init so url can be generated properly.

Reply:
yes, but there might be other files with that bad code...

Reply:
no sure of what you mean here.... There is only one callback for Lookup.

Reply:
yes, but other ui components using `Callback`?

Reply:
I think you had done most of them when you did #1397 - We can fix others, if so when needed.

Reply:
We need this fix ASAP for project....

Reply:
this takes like 5 minutes and should be parts of every fix - check, if the same bug is not anywhere else, at least in the same project

Reply:
I did and did not see any other, did you?

Reply:
> I did and did not see any other

great